pkg/serviceaccounts/oauthclient/oauthclientregistry.go

```diff
@@ -48,16 +47,16 @@ var (
 	}
 
 	emptyGroupKind       = schema.GroupKind{} // Used with static redirect URIs
-	routeGroupKind       = routeapi.SchemeGroupVersion.WithKind(routeKind).GroupKind()
-	legacyRouteGroupKind = legacy.GroupVersion.WithKind(routeKind).GroupKind() // to support redirect reference with old group
+	routeGroupKind       = schema.GroupKind{Group: "route.openshift.io", Kind: routeKind}
+	legacyRouteGroupKind = schema.GroupKind{Group: "", Kind: routeKind} // to support redirect reference with old group
```

Comment:
is this to support /oapi ? we don't support migration, so I think all these snowflakes can be removed now.

Reply:
No this keeps the SA as OAuth client stuff working without updating everyone's deployments.